build/llvm.yml

```diff
@@ -0,0 +1,231 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+trigger: none
+
+variables:
+  CCACHE_DIR: $(Pipeline.Workspace)/ccache
```

Comment:
Just curiosity. 
Is `CCACHE_DIR` related to https://ccache.dev/?
What is '**S**CCACHE' below? (in `SCCACHE_DIR`)

Reply:
sccache comes from https://github.com/mozilla/sccache. Note that Windows uses sccache while Linux and MacOS use ccache. I believe this was an intentional choice for compatibility, but I'm building off of infrastructure @idavis introduced for pyqir. That logic lives on here: https://github.com/qir-alliance/pyqir/blob/main/.github/workflows/ci.yml#L45. In that repo it also uses ccache for Linux and MacOS while Windows uses sccache.